test/e2e/resize_nodes.go

```diff
@@ -560,6 +582,121 @@ var _ = Describe("Nodes [Disruptive]", func() {
 					}
 				}
 			})
+
+			// What happens in this test:
+			// 	Network traffic from a node to master is cut off to simulate network partition
+			// Expect to observe:
+			// 1. Node is marked NotReady after timeout by nodecontroller (40seconds)
+			// 2. All pods on node are marked NotReady shortly after #1
+			// 3. Node and pods return to Ready after connectivivty recovers
+			It("All pods on the unreachable node should be marked as NotReady upon the node turn NotReady "+
+				"AND all pods should be mark back to Ready when the node get back to Ready before pod eviction timeout", func() {
+				By("choose a node - we will block all network traffic on this node")
+				nodeOpts := api.ListOptions{}
+				nodes, err := c.Nodes().List(nodeOpts)
+				Expect(err).NotTo(HaveOccurred())
+				filterNodes(nodes, func(node api.Node) bool {
+					return isNodeConditionSetAsExpected(&node, api.NodeReady, true)
+				})
+				if len(nodes.Items) <= 0 {
+					Failf("No node were found Ready: %d", len(nodes.Items))
+				}
+				node := nodes.Items[0]
+				podOpts := api.ListOptions{FieldSelector: fields.OneTermEqualSelector(client.PodHost, node.Name)}
```

Comment:
you're not actually creating pods right (or did i miss it)? so can you actually confirm this is a non zero number of pods